src/edge_proxy/settings.py

```diff
@@ -84,6 +84,8 @@ class LoggingSettings(BaseModel):
     log_format: LogFormat = LogFormat.GENERIC
     log_level: LogLevel = LogLevel.INFO
     log_event_field_name: str = "message"
+    colors: bool = True
```

Comment:
This hurts me, but I appreciate we're trying to maintain the same taxonomy as the uvicorn API... but * shudders in british *

Reply:
Addressed :)